index.html

```diff
@@ -39,4 +39,24 @@
     </div>
 </nav>
 
+<div class="hero__container">
+    <div>
+        <div class="hero__info">
+            <h1 class="hero__info__title">Always Track & Analyze Your Business Statistics To Succeed.</h1>
+            <span>A better way to manage your sales, team, clients & marketing</span>
+            <br>
+            <span>— on a single platform. Powerful, affordable & easy.</span>
+        </div>
+
+        <div class="hero__info__auth">
+            <input class="hero__info__auth__input" placeholder="Enter your email"/>
+            <button type="button" class="button--dark hero__info__auth__button">Get started</button>
+        </div>
```

Comment:
instead of a `div` use a `form` and `button type="submit"` and lets add some js to disable the submit